### PR TITLE
Add a /geyser ping command 

### LINKED ISSUE #3552: Add command /geyser ping
### What feature do you want to see added?

Add command `/geyser ping` ,then bedrock client use this command ping geyser.

### Are there any alternatives?

It seems that the existing simulation method cannot be used to replace this instruction

### PR DESCRIPTION
**Pull request by onebeastchris**
Implements https://github.com/GeyserMC/Geyser/issues/3552 to show the ping between the Bedrock client <-> Geyser server.
Huge thanks to rk_01 for the help!

Note: There are [two methods](https://github.com/CloudburstMC/Network/blob/develop/transport-raknet/src/main/java/org/cloudburstmc/netty/handler/codec/raknet/common/RakSessionCodec.java#L804-L810) in `RakSessionCodec` that can be used for this purpose - not fully sure which one would be the best to use here. Both seemed to have similar values during testing.

~~Requires https://github.com/GeyserMC/languages/pull/117 for the lang strings.~~

Closes #3552, closes https://github.com/GeyserMC/Geyser/issues/4379

~~Currently blocked by the cloud PR as it changes language strings.~~

### CODE REVIEW

**Comment by Konicai** [comment]
If Geyser's languages submodule is updated before the cloud PR is merged, I need to fix usages by the master branch of certain lang strings, before the submodule is updated. This is alright, I'm just saying it needs to be done first.

**Comment by onebeastchris** [comment]
This PR can totally wait until after cloud is merged, it's by no means urgent :)

**Review by rtm516 — APPROVED** [review]

**Pull request merged**
